R/measures.R

```diff
@@ -767,34 +767,34 @@ measureHAMLOSS = function(truth, response) {
   mean(truth != response)
 }
 
-#' @export subset01
+#' @export mlsubset01
 #' @rdname measures
 #' @format none
 subset01 = makeMeasure(id = "subset01", minimize = TRUE, best = 0, worst = 1,
   properties = c("multilabel", "req.pred", "req.truth"),
   name = "Subset-0-1 loss",
-  note = "Proportion of observations whose classes are not completely correctly predicted, 
+  note = "Proportion of observations whose classes are not completely correctly predicted,
   following the definition by Charte and Charte: https://journal.r-project.org/archive/2015-2/charte-charte.pdf",
   fun = function(task, model, pred, feats, extra.args) {
-    measureSUBSET01(getPredictionTruth.PredictionMultilabel(pred),
+    measureSubset01(getPredictionTruth.PredictionMultilabel(pred),
```

Comment:
Isn't our convention that the second part of the function name is uppercase? (Like measureMMCE etc.)?